packages/data-service/src/data-service.ts

```diff
@@ -749,7 +736,12 @@ export interface DataService {
 
   isConnected(): boolean;
 
-  isOperationCancelledError(error: Error): boolean;
+  /**
+   * If the error is an AbortError.
+   *
+   * @param error - The error to check.
+   */
+  isCancelError(error: Error): ReturnType<typeof isCancelError>;
```

Comment:
Similar to the original method, you might want to use `any` here, FWIW that's one of the legit cases for using it in the code and makes calling this method easier while still doing a correct assertion

```suggestion
  isCancelError(error: any): ReturnType<typeof isCancelError>;
```

Reply:
done 👍🏽